lcd_test/helpers_test.go

```diff
@@ -317,7 +317,7 @@ func InitializeTestLCD(t *testing.T, nValidators int, initAddrs []sdk.AccAddress
 	stakingDataBz = cdc.MustMarshalJSON(stakingData)
 	genesisState[staking.ModuleName] = stakingDataBz
 
-	genaccountsData := genaccounts.NewGenesisState(accs)
+	genaccountsData := genaccounts.GenesisState(accs)
```

Comment:
This name comes from the SDK so we can't do much here. But sounds like genaccounts.GenesisState is in fact returning a set of accounts, the name of the function is misleading.